include/Kitchen.hpp

```diff
@@ -0,0 +1,44 @@
+/*
+** EPITECH PROJECT, 2024
+** Plaza
+** File description:
+** No file there , just an epitech header example .
+** You can even have multiple lines if you want !
+*/
+
+#pragma once
+#include "Cooks.hpp"
+#include "SafeQueue.hpp"
+#include "Thread.hpp"
+#include "Recipe.hpp"
+#include <chrono>
+#include <map>
+
+struct coockersStruct {
```

Comment:
Can you rename this structure in pascal case, moreover I'm maintaining that this structure is useless